### PR TITLE
LPD-44860 Isolate problematic css with RTL converter

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by liferay-page-management** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:sf

File: modules/test/playwright/tests/layout-content-page-editor-web/sidebar.spec.ts [hunk 1]
```diff
@@ -170,6 +170,41 @@ test('Checks if sidebars are open or closed depending on Product Menu', async ({
...
+
+		await expect(
+			page.locator(
+				'.page-editor__sidebar__content.page-editor__sidebar__content--open.rtl'
+			)
+		).not.toBeVisible();
+
+		// Return to initial language
+
+		await page.goto(`/en/web${site.friendlyUrlPath}${layout.friendlyURL}`);
```
Comment (victorg1991) [inline modules/test/playwright/tests/layout-content-page-editor-web/sidebar.spec.ts:204]: This should be `layout.friendlyUrlPath`

**Comment by veroglez** [comment]
ci:test:page-management-playwright

**Comment by victorg1991** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**